subsys/dfu/dfu_target/Kconfig

```diff
@@ -79,6 +79,16 @@ config DFU_TARGET_FULL_MODEM
 	  Enable support for full updates to the modem firmware.
 	  Note that this requires an external flash.
 
+config DFU_TARGET_FULL_MODEM_EXT_FLASH_SIZE
+	hex "Flash size required for storing full modem image"
+	depends on DFU_TARGET_FULL_MODEM
+	default 0x400000 if SOC_NRF9160
```

Comment:
Can't we get this value from devicetree?

Reply:
The size of the flash is anyway dependent on the board, not the SOC.

Reply:
This is the required flash size for the full modem update, 4MB.

Reply:
Ah, my bad. Why does this have a prompt then? Would a user ever change it?

Reply:
yeah. i can't think of a reason why the user would want to change it.

Reply:
would you prefer that the prompt be removed?

Reply:
Yes. Kconfig options that will not be modified by users should not have prompts.